private/dependency_tree_parser.bzl

```diff
@@ -121,7 +122,7 @@ def _generate_target(
     #
     is_dylib = False
     if packaging == "jar":
-        target_import_string.append("\tjars = [\"%s\"]," % artifact_path)
```

Comment:
This is generating the jvm_import statement right?  I don't see where jvm_import can take `jar` instead of `jars` ? https://github.com/bazelbuild/bazel/blob/master/src/main/starlark/builtins_bzl/common/java/java_import.bzl#L264

Reply:
nvm, this is for a different java_import that we define